demo/demo-local-service-registry/local-registry-client/pom.xml

```diff
@@ -0,0 +1,54 @@
+<?xml version="1.0"?>
```

Comment:
Can you move these codes to samples folder？ Java-chassis demo folder is for integration tests . We are planning to change it but got no time to deal this. 
And add a README.md for this sample project to learn what it is do and how to run this sample.

Reply:
ok..i will move and raise PR.